Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -320,7 +320,7 @@ - (void)_setupVisibilityPropagationViewForGPUProcess
     if (!_visibilityPropagationInteractionForGPUProcess && !_visibilityPropagationInteractionForGPUProcess) {
         SEL selector = NSSelectorFromString(@"createVisibilityPropagationInteraction");
         if ([_page->process().extensionProcess().get() respondsToSelector:selector])
```

Comment:
But then this check is now out of sync.

Reply:
Ah, good catch! Will fix.

Thanks for reviewing!